RevenueCatUI/Views/FooterView.swift

```diff
@@ -167,8 +167,10 @@ private struct RestorePurchasesButton: View {
 
     var body: some View {
         AsyncButton {
-            _ = try await self.purchaseHandler.restorePurchases()
-            self.displayRestoredAlert = true
+            let success = try await self.purchaseHandler.restorePurchases().success
+            if success {
+                self.displayRestoredAlert = true
+            }
```

Comment:
this looks good to me as a stopgap, but I think we should try to get copy for saying something about "no purchases found" and localize it. 
No-op is kind of an awkward experience to me

Reply:
Filed a Linear for this.